ui/lib/pki/addon/routes/overview.js

```diff
@@ -7,28 +7,15 @@ export default class PkiOverviewRoute extends Route {
   @service auth;
   @service store;
 
-  get win() {
-    return this.window || window;
-  }
-
   hasConfig() {
     // When the engine is configured, it creates a default issuer.
     // If the issuers list is empty, we know it hasn't been configured
-    const endpoint = `${this.win.origin}/v1/${this.secretMountPath.currentPath}/issuers?list=true`;
-
-    return this.auth
-      .ajax(endpoint, 'GET', {})
+    return this.store
+      .query('pki/issuer', { backend: this.secretMountPath.currentPath })
       .then(() => true)
       .catch(() => false);
   }
```

Comment:
I noticed this while debugging and thought it would be easier to query from the store. If this needed to make an ajax request directly let me know and I will revert.

Reply:
Thinking out loud: Is there any situation where the store wouldn't have this model yet? Maybe navigating to this route such that pki/issuer isn't hit yet? Though I believe the default behavior is if it's not in the store then it will just fire off the network request. If this is the case, then what you did makes sense to me.

Reply:
Yep so the `query` method always makes a request to the server whereas I think `findRecord` will check the store first and then go to the server if not found.